iBudget/fund/views.py

```diff
@@ -30,11 +31,15 @@ def show_fund(request):
             HttpResponse object.
     """
     user = request.user
+    icon_if_none = \
+        'https://s3.amazonaws.com/family-finance-tracker-static/standard_fund/funds.png'
```

Comment:
This is a bad idea to have such hardcoded URLs in code. Ideally it should be built with parts that are constants. For instance, `icon_if_none = AWS_S3_URL + STANDARD_FUNDS_LOCATION + icon_file_name` where `AWS_S3_URL = https://s3.amazonaws.com`, `STANDARD_FUNDS_LOCATION = /family-finance-tracker-static/standard_fund/` and `icon_file_name=funds.png`.

Reply:
thanks, I will rewrite it like you said.